api/spec/controllers/spree/api/v1/checkouts_controller_spec.rb

```diff
@@ -310,6 +340,12 @@ module Spree
         api_put :advance, id: order.to_param, order_token: order.guest_token
         expect(json_response['id']).to eq(order.id)
       end
+
+      def send_request(params = {})
```

Comment:
Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. You can also write as `send_request(*)` if you want the method to accept any arguments but don't care about them.